src/utils/featureMap.ts

```diff
@@ -0,0 +1,12 @@
+export enum FeatureName {
+  Default = 0,
+  TokenNameRestructure
+}
+
+const FeatureMap: Record<string, FeatureName> = {
+  '2.1.0': FeatureName.TokenNameRestructure
+};
+
+export const getFeatureNameFromSdkVersion = (sdk: string): FeatureName => {
+  return FeatureMap[sdk] ?? FeatureName.Default;
+};
```

Comment:
This does not look correct.

Since the FeatureMap has sdk version as keys, each sdk version can have only 1 feature. But we need support for multiple feature per sdk version.

A better approach would be to inverse the map.

```ts
const FeatureMap: Record<FeatureName, {from: string; to: string?; }> = {
    [FeatureName.TokenNameRestructure]: {.from: '2.1.0', to: undefined }
  };

export const isFeatureEnabled = (featureName, sdkVersion) => boolean
```

Reply:
If we use the isFeatureEnabled then we would have to loop through every feature to check which ones are enabled. How about I just make the map have an array of FeatureNames something like..

```ts
const FeatureMap: Record<string, Array<FeatureName>> = {
  '2.1.0': [FeatureName.TokenNameRestructure]
};

export const getFeatureListFromSdkVersion = (
  sdk: string
): Array<FeatureName> => {
  return FeatureMap[sdk] ?? [];
};
```